integration-tests/funqy-google-cloud-functions/README.md

```diff
@@ -0,0 +1,77 @@
+# Google Cloud Functions - Funqy Binding
+
+This integration test has no automated test, it needs to be launched manually.
+
+## Build the artifact
+
+First, you need to log in to Google Cloud:
+
+```shell script
+gcloud auth login
+```
+
+Then you need to use Maven to build the artifact and put it in a directory where it will be the only file.
+
+```shell script
+mkdir deployment
+mvn clean package
+cp target/quarkus-integration-test-funqy-google-cloud-functions-999-SNAPSHOT-runner.jar deployment/
+```
+
+Finally, you need to use `gcloud` to deploy the function to Google Cloud. The `gcloud` command is different for HttpFunction and
+Background function so the set of instruction differs for each.
+
+This example contains multiple Funqy functions, if you want to test a different function that the one defined inside
+your `application.properties`, you can use the `--set-env-vars` option of `gcloud` to define the name of the function via the 
+`QUARKUS_FUNQY_EXPORT` environment variable.
+
+## HTTP function
+To deploy the HttpFunction, you can use the following `gcloud` command:
+
+```shell script
+gcloud beta functions deploy quarkus-funqy-http --entry-point=io.quarkus.funqy.gcp.functions.FunqyHttpFunction \
```

Comment:
If possible this deployment directory should be in target/ and created and setup by the extension.

Reply:
+1 to put this in target, I'll update the docs